mic_vad_streaming/mic_vad_streaming.py

```diff
@@ -217,7 +217,7 @@ def main(ARGS):
     parser.add_argument('-d', '--device', type=int, default=None,
                         help="Device input index (Int) as listed by pyaudio.PyAudio.get_device_info_by_index(). If not provided, falls back to PyAudio.get_default_device().")
     parser.add_argument('-r', '--rate', type=int, default=DEFAULT_SAMPLE_RATE,
-                        help=f"Input device sample rate. Default: {DEFAULT_SAMPLE_RATE}. Your device may require 44100.")
+                        help="Input device sample rate. Default: {DEFAULT_SAMPLE_RATE}. Your device may require 44100.")
```

Comment:
@Mhkgrover Could you please explain why you change that? I think `f""` is required for `{DEFAULT_SAMPLE_RATE}` to be displayed, isn't it?

Reply:
Running `mic_vad_streaming.py` results with `SyntaxError` that's caused by that `f""`. I've changed it to: `"Input device sample rate. Default: {0}. Your device may require 44100.".format(DEFAULT_SAMPLE_RATE)`. Works like that.